Makefile

```diff
@@ -212,29 +212,34 @@ all-local: $(ALL) ## Build all artifacts in the build tree
 .PHONY: all-local
 
 $(ISO_ROOT)/bootstrap.sh: scripts/bootstrap.sh.in $(ISO_ROOT)/product.txt
-	mkdir -p $(shell dirname $@)
+	mkdir -p $(@D)
 	rm -f $@
 	sed s/@VERSION@/$(shell source $(ISO_ROOT)/product.txt && echo $$SHORT_VERSION)/g < $< > $@ || (rm -f $@; false)
 	chmod a+x $@
 
 $(ISO_ROOT)/salt/%: salt/%
-	mkdir -p $(shell dirname $@)
+	mkdir -p $(@D)
 	rm -f $@
 	cp -a $< $@
 
+$(ISO_ROOT)/pillar/metalk8s.sls: pillar/metalk8s.sls.in $(ISO_ROOT)/product.txt
```

Comment:
Minor: this could be merged with the rule below for `$(ISO_ROOT)/pillar/%.sls`, but given #708 it's not worth it :)

Reply:
That's what I thought, and we have another `$(ISO_ROOT)/pillar/repositories.sls` which does not follow the same rule, so I was not sure how to write the pattern :/